test_project/settings.py

```diff
@@ -186,7 +163,25 @@
         'cities_light': {
             'handlers':['console'],
             'propagate': True,
-            'level':'DEBUG',
+            'level':'WARNING',
         },
     }
 }
+
+if os.environ.get('CI', False):
+    CITIES_LIGHT_TRANSLATION_LANGUAGES=['fr', 'ru']
```

Comment:
Yes, that is long but it can help to spot some problems (performance for example). Also we've seen an index problem with postgresql due to too long alternate_names field:

```
django.db.utils.OperationalError: index row size 2880 exceeds maximum 2712 for index "cities_light_city_610a2ab0"
HINT:  Values larger than 1/3 of a buffer page cannot be indexed.
Consider a function index of an MD5 hash of the value, or use full text indexing.
```